src/C4SoundSystem.cpp

```diff
@@ -347,10 +362,10 @@ auto C4SoundSystem::NewInstance(const char *filename, const bool loop,
 	return &inst;
 }
 
-std::string C4SoundSystem::PrepareFilename(const char *const filename)
+std::string C4SoundSystem::PrepareFilename(const char *const filename, std::string fallbackFileExtension)
```

Comment:
Passing `std::string` by value is only useful when a copy is needed, use `std::string_view` instead.